README.md

```diff
@@ -13,7 +13,8 @@ Things you may want to cover:
 * System dependencies
 
 * Configuration
-
+To ecnrypt the API key run: `gpg --output config/application.yml.asc --symmetric config/application.yml`
+To decrypt: gpg --output config/application.yml --decrypt config/application.yml.asc
```

Comment:
Since this is for future teams, perhaps include that the decryption key will be provided by the course staff in case they get confused